taxi/views.py

```diff
@@ -84,6 +126,28 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+
+    def get_success_url(self):
```

Comment:
Here you don't need to set get_success_url because get_absolute_url is defined in your model - this should redirect to DriverDetail under the hood